ballerina/Dependencies.toml

```diff
@@ -195,9 +195,7 @@ org = "ballerina"
 name = "lang.runtime"
 version = "0.0.0"
 dependencies = [
-	{org = "ballerina", name = "jballerina.java"},
-	{org = "ballerina", name = "lang.array"},
-	{org = "ballerina", name = "lang.value"}
+	{org = "ballerina", name = "jballerina.java"}
```

Comment:
A tangent issue, is this change happening only for `-x test` build? If so we need to run this commit task only is tests are enabled or not?

Reply:
Nope, I did a full build.

Reply:
I noticed this behaviour when building without tests, the dependencies get updated and reverts again. This is adding unnecessary automatic commits.

Reply:
Cool.